themes/bootstrap3/templates/Helpers/copy-to-clipboard-button.phtml

```diff
@@ -0,0 +1,29 @@
+<p>
+    <button type="button" id="copyToClipboard" class="btn btn-primary hidden" role="button" tabindex="0"><?=$this->translate("copy_to_clipboard_button_label")?></button>
```

Comment:
Do you think there might be a scenario where we have more than one of these on the page at the same time? If so, it shouldn't have a hard-coded ID. Should we add a static counter to the view helper class and number them for uniqueness?

Reply:
It could be possible and adding a counter does not harm, so I've done it. Thanks for that idea @demiankatz